### ISSUE TITLE
Just a remark on mobile SDK

### ISSUE THREAD

**Issue by FabriceSalvaire** [maplibre/maplibre.github.io#15]
I don't want to judge... But I did some investigations on the repositories of X and I realized after one hour that the current mobile SDK is in fact based on a binary blob that require an account to be downloaded. Well, I probably missed a paragraph on the website when I clicked on the GitHub icon.

Thus, to clarify things, I would suggest specifying (reword) that the original "GL Native" is probably in an GitHub archived(*) state.

In other words, I concluded the former is discontinued and the newer is just a "facade".  For me, it is a bit different that a "switch to a non-OSS licence" as it is written in the README.

(*)  I noticed some repositories are tagged as **banished**

**Comment by nyurik** [comment]
Hi @FabriceSalvaire, I read your post twice... and still couldn't understand what you mean.  Can you clarify? Thanks!

**Comment by maxammann** [comment]
As there was no further response I would close this.
